docs/user-guide/knowledge-distillation.rst

```diff
@@ -0,0 +1,362 @@
+.. include:: /content/nemo.rsts
+
+SFT with Knowledge Distillation
```

Comment:
Spell out acronym. This helps us with SEO.

Supervised Fine-Tuning (SFT) with Knowledge Distillation